virttest/qemu_devices/qcontainer.py

```diff
@@ -2896,6 +2896,20 @@ def throttle_group_define_by_params(self, group_params, name):
         props = json.loads(group_params.get("throttle_group_parameters", "{}"))
         return QThrottleGroup(name, props)
 
+    def thread_context_define_by_params(self, params, name):
+        """
+        Create thread-context object from params.
+        """
+        tc_params = Params()
+        prefix = 'vm_thread_context_'
+        for key in list(params.keys()):
```

Comment:
```suggestion
        for key in params:
```
Looks like we could call the magic method `__iter__` of the params object(`class Params(IterableUserDict)`).

Reply:
Updated now, thanks!